### PR TITLE
Added ipsec client 3.2.0 docs

### PR DESCRIPTION
**Pull request by BenMatase**

### CODE REVIEW

**Comment by BenMatase** [comment]
BTW @MichaelBaj I tried doing the docker compose to locally test and I was hitting an issue saying that the `node` version needed to be >= 14. then when I tweaked that, it seemed like it was hanging when bringing up the container. Not sure if the process has changed or something, but I couldn't locally verify these changes.

**Comment by BenMatase** [comment]
Hit some OOM on last run https://github.com/128technology/docs/runs/5305847549?check_suite_focus=true

File: docs/plugin_ipsec_client.md [hunk 1]
```diff
@@ -84,6 +84,43 @@ exit
...
+                exit
+            exit
+        exit
+
+        tunnel-monitor-nat-network 10.128.128.0/28
+    exit
+exit
+```
+
+* `enabled` - Allows you to switch tunnel monitoring on and off for a `remote`
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_ipsec_client.md:117]: period after remote.

File: docs/plugin_ipsec_client.md [hunk 2]
```diff
@@ -84,6 +84,43 @@ exit
...
+* `address` - The ip or hostname to send traffic to. This address must be reachable after traversing the tunnel.
+* `timeout` - Duration (in seconds) within which to reach the destination. Each attempt will be made in this duration / `max-retries` interval.
+* `max-retries` - Number of consecutive missed ICMP ping responses from the destination within the interval before deciding that the tunnel is unhealthy.
+* `interval` - Duration (in seconds) of how often to perform an ICMP probe test to the probe-address.
+* `tunnel-monitor-nat-network` - The subnet to originate traffic from. The correspnoding ingress KNI's fourth octet will be used. By default, the subnet `10.128.128.0/28` is used.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/plugin_ipsec_client.md:122]: The subnet where traffic originates.
The corresponding ingress KNI's fourth octet is used. (corresponding was misspelled, and changed will be to is.)

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]

**Comment by BenMatase** [comment]
Hit OOM again, so retriggered

**Review by agrawalkaushik — APPROVED** [review]

**Review by jeffmart-jnpr — APPROVED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**